lib/webauth.js

```diff
@@ -169,7 +169,7 @@ AuthResponse.prototype = {
     let result = false;
     for (const type of Object.keys(this.categories)) {
       const authCategoryResult = this.categories[type];
-      if (!(typeof authCategoryResult) === "object") {
+      if (typeof authCategoryResult !== "object") {
```

Comment:
This expression was of type boolean, but it was compared to an expression of type string.